backend/app/models/Query.scala

```diff
@@ -0,0 +1,24 @@
+package models
+
+import play.api.libs.json._
+
+
```

Comment:
Remove blank line (or add another for consistency in other files after the imports)